miden-lib/asm/eoa/basic.masm

```diff
@@ -0,0 +1,32 @@
+use.miden::sat::account
+use.miden::sat::tx
+
+#! Authenticate a transaction using Falcon 
+#! WIP - FALCON VERIFICATION IS MISSING, NOT SURE IF HASHING IN CORRECT ORDER
+#! Stack: []
+#! Output: []
+#!
+#! - ASSET is the non-fungible asset of interest.
+#! - tag is the tag to be included in the note.
+#! - RECIPIENT is the recipient of the note.
+export.auth_tx
```

Comment:
I would probably name this much more explicitly - for example, something like `auth_tx_rpo_falcon512` (`rpo_falcon512` is the name of the signature scheme).

Also, I think we should mention in the comment that this procedure assumes that Falcon public key is stored in storage slot 0.